rust/gui-client/src-tauri/src/client/ipc/windows.rs

```diff
@@ -11,7 +12,7 @@ pub(crate) type IpcStream = named_pipe::NamedPipeClient;
 /// This is async on Linux
 #[allow(clippy::unused_async)]
 pub(crate) async fn connect_to_service() -> Result<IpcStream> {
-    let path = firezone_headless_client::ipc::platform::pipe_path();
+    let path = ipc::platform::pipe_path(ipc::ServiceId::Prod);
```

Comment:
Even though it isn't the same, would it make sense to merge `pipe_path` and `sock_path` to just `ipc_path`?